modules/internal/ChapelTuple.chpl

```diff
@@ -19,16 +19,68 @@
  */
 
 /*
-Predefined Functions on Tuples.
-
 Tuples are a predefined structured type in Chapel. They are specified
 in the Tuples chapter of the Chapel Language Specification.
 This page lists the predefined functions on tuples.
 They are always available to all Chapel programs.
 
-Besides the functions defined here, the Chapel Language specification
-defines other operations available on tuples: indexing, iteration,
-assignment, and unary, binary, and relational operators.
+The tuple type supports a wide variety of operators, some of which can only
```

Comment:
Thanks for the updated examples and descriptions!

In the spirit of centralizing the docs for built-ins in the language spec section (e.g., https://github.com/chapel-lang/chapel/issues/18027 and https://github.com/chapel-lang/chapel/issues/16814), do you think it would be prudent to move these updates to that location? I am thinking of the specific portions here: `language/spec/tuples.html#unary-operators` and here: `language/spec/tuples.html#binary-operators`

Reply:
That's a good point, I didn't realize we already had detailed docs for the operators!

I'll move the examples to those language-spec sections since they'd probably end up there anyway when the Tuples portion of #18027 is completed.